src/templates/events/past-events-template.jsx

```diff
@@ -15,9 +15,7 @@ const PastEventsTemplate = ({data}) => {
   })
 
   return (
-    <PageContent width="95%" maxWidth="800px">
-
-      <Title>Event Archive</Title>
+    <PageContent width="95%" maxWidth="800px" title="Event Archive">
```

Comment:
`width="95"` is the default. let's remove unnecessary props like this.

Reply:
ah yes, thank you!